test/models/user_test.rb

```diff
@@ -11,6 +11,26 @@ class UserTest < ActiveSupport::TestCase
     refute person.valid?
   end
 
+  test "encrypts openai_key" do
+    user = users(:keith)
+    old_openai_key = user.openai_key
+    old_cipher_text = user.ciphertext_for(:openai_key)
+    user.update(openai_key: "new one")
```

Comment:
Personal preference, just in case test starts failing we can more easily spot if related to the test itself or its setup.

```suggestion
    user.update!(openai_key: "new one")
```